consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -483,120 +483,111 @@ impl<
         }
     }
 
-    fn attempt_complete_cur_slot(&mut self) {
-        if let Some(ext_vals) = self.scp.get_externalized_values(self.current_slot_index) {
```

Comment:
I basically moved the `if let Some(...)` out of this method. I think this feels simpler and more "declarative".